Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1477,6 +1477,8 @@ static Ref<CSSValue> fontVariantAlternatesPropertyValue(FontVariantAlternates al
 {
     if (alternates.isNormal())
         return CSSValuePool::singleton().createIdentifierValue(CSSValueNormal);
+    if (alternates.values().historicalForms)
+        return CSSValuePool::singleton().createIdentifierValue(CSSValueHistoricalForms);
```

Comment:
Does this work correctly if you have `font-variant-alternates: historical-forms stylistic(foo)`?

Reply:
Probably not, I'll add coverage and fix as needed. I didn't realize this was valid syntax. However, I just checked the spec and you're right.

Reply:
Fixed.